packages/smart-wallet/src/smartWallet.js

```diff
@@ -1052,6 +1057,19 @@ export const prepareSmartWallet = (baggage, shared) => {
          * @throws if the seat can't be found or E(seatRef).tryExit() fails.
          */
         async tryExitOffer(offerId) {
+          const { facets } = this;
+          const amts = await facets.payments
+            .tryReclaimingWithdrawnPayments(offerId)
+            .catch(e => {
+              facets.helper.logWalletError(
+                'recovery failed reclaiming payments',
+                e,
+              );
+              return [];
+            });
+          if (amts.length > 0) {
+            facets.helper.logWalletError('reclaimed', amts, 'from', offerId);
```

Comment:
not a blocker but this is success, not an error.
```suggestion
            facets.helper.logWalletInfo('reclaimed', amts, 'from', offerId);
```

Reply:
Thank you, @turadg. I wondered why it was an error, but didn't get as far as identifying that it didn't need to be.